minio/minio
# etcd unavailable start of minio or during run

## Description
Fixes #9788

## Motivation and Context
etcd dependent operations like `mc admin policy add ...` fail during execution.

## How to test this PR?
Case 1.
Start etcd, minio in that order. Minio should configure the etcd endpoint. Stop etcd and run `mc admin policy add ...` to check.
Case 2.
Start minio with etcd endpoint. Run `mc admin policy add ...` to check. Restart etcd and check again.

## Types of changes

## Checklist:
